README.md

```diff
@@ -20,7 +20,7 @@ We recommend using `nvm`, the [Node Version Manager](https://github.com/creation
 
 # Full Documentation
 
-View our [Javascript SDK Documentation](https://www.bitgo.com/api/v2).
+View our [API Documentation](https://www.bitgo.com/api/v2).
```

Comment:
Could you update this to https://www.bitgo.com/api/v2/slate while you're making changes here?